src/test/java/at/ac/tuwien/kr/alpha/common/terms/TestMinusTerm.java

```diff
@@ -41,14 +41,14 @@
 
 	@Test
 	public void testNormalizeVariablesNoVariable() {
-		MinusTerm m2 = MinusTerm.getInstance(ConstantTerm.getInstance(2));
+		Term m2 = MinusTerm.getInstance(ConstantTerm.getInstance(2));
```

Comment:
I do not understand the rationale behind the changes in this file.

Reply:
Rationale: if an ArithmeticTerm or MinusTerm is ground `getInstance` now returns the corresponding ConstantTerm, hence `MinusTerm.getInstance(..)` now only returns `Term` instead of `MinusTerm`. In the case here it returns a `ConstantTerm`.